tools/setup-prebuild-provider-folder.sh

```diff
@@ -48,8 +48,9 @@ cd packages
 for repo in $REPO_NAMES; do
     echo "Adding $repo"
     
-    gh repo clone "hashicorp/$repo"
+    gh repo clone "hashicorp/$repo" &
```

Comment:
Does the for-loop wait for this to complete? Or does this race with the `rm` commands below?

Reply:
yeah, added a wait now that waits for all open jobs

Reply:
nice! TIL that `wait` helps with this 💡